src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/DefaultProjectSnapshot.cs

```diff
@@ -38,11 +38,15 @@ public DefaultProjectSnapshot(ProjectState state)
 
         public HostProject HostProject => State.HostProject;
 
-        public override bool IsInitialized => WorkspaceProject != null;
-
         public override VersionStamp Version => State.Version;
 
-        public override Project WorkspaceProject => State.WorkspaceProject;
+        public override IReadOnlyList<TagHelperDescriptor> TagHelpers => State.TagHelpers;
+
+#pragma warning disable CS0672 // Member overrides obsolete member
+        public override bool IsInitialized => throw new NotImplementedException();
+
+        public override Project WorkspaceProject => throw new NotImplementedException();
+#pragma warning restore CS0672 // Member overrides obsolete member
```

Comment:
I assume this is just for the design-pr part 😁

Reply:
> * Obsoleted Workspace related members from the `ProjectSnapshot`in order to ensure that current LiveShare continues to work. Once LiveShare has been incorporated into Razor we can remove these members from the project snapshot.

Reply:
Do we have an issue tracking bringing LiveShare over to Razor? If so we should add this to that so this isn't forgotten.

Reply:
I'm working on that simultaneously and have already made the changes locally. I hope to have a PR out for it today 😄 

As for an issue to track bringing LiveShare over to Razor, we have one in aspnet/Razor.LiveShare#47